pkgs/applications/networking/browsers/chromium/patches/no-build-timestamps.patch

```diff
@@ -0,0 +1,15 @@
+--- chromium-70.0.3538.67/build/compute_build_timestamp.py.orig	2018-11-02 16:00:34.368933077 +0200
++++ chromium-70.0.3538.67/build/compute_build_timestamp.py	2018-11-03 18:35:20.542943107 +0200
+@@ -94,6 +94,12 @@
+       'build_type', help='The type of build', choices=('official', 'default'))
+   args = argument_parser.parse_args()
+ 
++  # I don't trust LASTCHANGE magic, and I definelly want something deterministic here
++  SOURCE_DATE_EPOCH = os.getenv("SOURCE_DATE_EPOCH", None)
++  if SOURCE_DATE_EPOCH is not None:
```

Comment:
I think we can be sure that ```SOURCE_DATE_EPOCH is not None``` for Nix builds
If it is not so because of some bug, it would be better ```assert``` and crash here than silently falling through to the old code.

Reply:
Will add assert on next iteration ;)
I have some plans to check why some third party stuff listed as build inputs, but build vendored unconditionally (icu, ffmpeg, webrtc, some codecs).
Also I curious, if possible to kick angle and/or skia to own derivations (not sure if it possible w/o huge patching)

Reply:
@volth added assert for missing SOURCE_DATE_EPOCH as suggested